module/zfs/dbuf.c

```diff
@@ -48,6 +48,97 @@
 #include <sys/callb.h>
 #include <sys/abd.h>
 
+kstat_t *dbuf_ksp;
+
+typedef struct dbuf_stats {
+	/*
+	 * Various statistics about the size of the dbuf cache.
+	 */
+	kstat_named_t cache_count;
+	kstat_named_t cache_size_bytes;
+	kstat_named_t cache_size_bytes_max;
+	/*
+	 * Statistics regarding the bounds on the dbuf cache size.
+	 */
+	kstat_named_t cache_target_bytes;
+	kstat_named_t cache_lowater_bytes;
+	kstat_named_t cache_hiwater_bytes;
+	kstat_named_t cache_max_bytes;
```

Comment:
Let's remove `cache_max_bytes` from the stats.  This value is already readable from the `dbuf_cache_max_bytes` module option so we're not losing any information.  But more importantly I think it's too easy to get confused with the `cache_size_bytes_max` since the names are almost identical.  Best to remove it and avoid any confusion entirely.